pyanaconda/ui/gui/spokes/advstorage/nvdimm.py

```diff
@@ -0,0 +1,134 @@
+        self._conditionNotebook = self.builder.get_object("conditionNotebook")
+
+    def refresh(self):
+        self._sectorSizeSpinButton.set_value(DEFAULT_SECTOR_SIZE)
+        if self.namespaces:
+            self._devicesLabel.set_text("%s" % ", ".join(self.namespaces))
+        else:
+            msg = CN_("GUI|Advanced Storage|NVDIM", "No device to be reconfigured selected.")
+            self._infoLabel.set_text(msg)
+            for widget in [self._sectorSizeSpinButton, self._okButton, self._startButton,
+                           self._sectorSizeLabel]:
+                widget.set_sensitive(False)
+
+    def run(self):
+        rc = self.window.run()
+        self.window.destroy()
+        if self._update_devicetree:
+            self.storage.devicetree.reset()
+            try_populate_devicetree(self.storage.devicetree)
+        return rc
+
+    @property
+    def sector_size(self):
+        return self._sectorSizeSpinButton.get_value_as_int()
+
+    def on_start_clicked(self, *args):
+        self._conditionNotebook.set_current_page(PAGE_ACTION)
+
+        for widget in [self._startButton, self._cancelButton, self._sectorSizeSpinButton,
+                       self._okButton]:
+            widget.set_sensitive(False)
+
+        self._waitSpinner.start()
+
+        threadMgr.add(AnacondaThread(name=constants.THREAD_NVDIMM_RECONFIGURE, target=self._reconfigure,
+                                     args=(self.namespaces, "sector", self.sector_size)))
+
+    def _reconfigure(self, namespaces, mode, sector_size):
+        for namespace in namespaces:
+            if namespace in nvdimm.namespaces:
+                log.info("nvdimm: reconfiguring %s to %s mode", namespace, mode)
+                try:
+                    nvdimm.reconfigure_namespace(namespace, mode, sector_size=sector_size)
+                    self._update_devicetree = True
+                except Exception as e:  # pylint: disable=broad-except
+                    self._error = "%s: %s" % (namespace, str(e))
+                    log.error("nvdimm: reconfiguring %s to %s mode error: %s",
+                              namespace, mode, e)
+                    break
+            else:
+                log.error("nvdimm: namespace %s to be reconfigured not found", namespace)
+        self._after_reconfigure()
+
+    @gtk_action_wait
+    def _after_reconfigure(self):
+        # When reconfiguration is done, update the UI.  We don't need to worry
+        # about the user escaping from the dialog because all the buttons are
+        # marked insensitive.
+        self._waitSpinner.stop()
+
+        if self._error:
+            self.builder.get_object("deviceErrorLabel").set_text(self._error)
+            self._error = None
+            self._okButton.set_sensitive(True)
+            self._conditionNotebook.set_current_page(PAGE_RESULT_ERROR)
+        else:
+            self._okButton.set_sensitive(True)
```

Comment:
I think you can merge these two calls of `self._okButton.set_sensitive(True)` outside of that if-else branch.  Of course if that shouldn't be `False` in one branch.